packages/sanity/src/structure/panes/document/documentPanel/DocumentPanel.tsx

```diff
@@ -136,6 +126,22 @@ export const DocumentPanel = function DocumentPanel(props: DocumentPanelProps) {
 
   const showInspector = Boolean(!collapsed && inspector)
 
+  const banners = useMemo(() => {
+    if (activeView.type !== 'form' || isPermissionsLoading || !ready) return null
+
+    return (
+      <>
+        <PermissionCheckBanner
+          granted={Boolean(permissions?.granted)}
+          requiredPermission={requiredPermission}
+        />
+        <ReferenceChangedBanner />
+        <DeprecatedDocumentTypeBanner />
+        <DeletedDocumentBanners />
```

Comment:
This had a `{!isDeleting && isDeleted && (` check , we don't need that anymore?

Reply:
This is in [here now](https://github.com/sanity-io/sanity/pull/7371/files/e621f76022fc08ac0f59c34f4d81a753563b4696#diff-23d9f184f5425ff6d768a974daa289f3b5df66ed8c6b2f2b20bd89fa109e02a0R47)

Reply:
missed it, thanks!